---
title: EecSeq Bioinformatics
summary: 
tags:
  - EecSeqBio
date: "2016-04-27T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

image: 
  caption: 
  focal_point: Smart

links:
- icon: github
  icon_pack: fab
  name: code
  url: https://github.com/MarineEvoEcoLab/EecSeq
- icon: atom
  icon_pack: fas
  name: NSF-2016160
  url: https://nsf.gov/awardsearch/showAward?AWD_ID=2016160
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
#slides: example
---
[![NSF-2016160](https://img.shields.io/badge/NSF-2016160%20-blue)](https://nsf.gov/awardsearch/showAward?AWD_ID=2016160 )

To develop EecSeq into an exome capture method for any organism, the accompanying bioinformatics pipeline needs to be capable of de novo assembly of exon loci directly from captured genomic reads. Assembling exon loci is a new bioinformatic challenge as traditional exome capture relies on designed probes from genomic and/or transcriptomic data. Accurate *de novo* assembly is critical to population level inference derived from reduced representation data sets, and errors, artifacts, and biases in assembly are still problematic in both RADseq and RNAseq. Leveraging the chromosome-level assembly of the eastern oyster genome, we are developing two complementing de novo assembly methods for EecSeq: one utilizing only captured genomic reads and a second hybrid method that will utilize sequences from the cDNA probes (when sequenced) and captured genomic reads. The final output will be an open source bioinformatics pipeline for EecSeq.

![image](https://github.com/MarineEvoEcoLab/Lab_Website/assets/4837703/a7ddb643-4c8a-45b4-a04a-753b228b8947)

